src/client/startup.js

```diff
@@ -147,6 +147,8 @@ sand.globalFunctions = {
 					}
 				}
 			});
+			// running this after the ajax request in case the user has analytics blocked
```

Comment:
nice.

in that case, will it work if you surround the function call with a try catch statement?
just throw the error away afterward.

That way if anyone decides to add code after here under the mistaken assumption that it will always work, they'll be able to.

and also rewrite the comment to reflect this (i.e. "Occasionally this function will throw an error when the client has blocked google analytics")